src/loaders/ncbi_downloader/ncbi_downloader.py

```diff
@@ -1,74 +1,76 @@
 """
-PROTOTYPE - Download genome files from NCBI FTP server.
-
-usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver
-                          RELEASE_VER [--root_dir ROOT_DIR] [--source SOURCE] [--threads THREADS]
-                          [--overwrite]
+usage: ncbi_downloader.py [-h] --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...] --release_ver {202,207,214,80,83,86,89,95}
+                          [--kbase_collection {GTDB}] [--root_dir ROOT_DIR] [--source {NCBI}] [--threads THREADS] [--overwrite]
                           [--exclude_name_substring EXCLUDE_NAME_SUBSTRING [EXCLUDE_NAME_SUBSTRING ...]]
 
+PROTOTYPE - Download genome files from NCBI FTP server.
+
 options:
   -h, --help            show this help message and exit
 
 required named arguments:
   --download_file_ext DOWNLOAD_FILE_EXT [DOWNLOAD_FILE_EXT ...]
-                        Download only files that match given extensions.
-  --release_ver RELEASE_VER
+                        Download only files that match given extensions
+  --release_ver {202,207,214,80,83,86,89,95}
                         GTDB release version
 
 optional arguments:
-  --root_dir ROOT_DIR   Root directory.
-  --source SOURCE       Source of data (default: GTDB)
-  --threads THREADS     Number of threads. (default: half of system cpu count)
-  --overwrite           Overwrite existing files.
+  --kbase_collection {GTDB}
+                        Create a collection and link in data to that collection from the overall source data dir (default: GTDB)
+  --root_dir ROOT_DIR   Root directory (default: /global/cfs/cdirs/kbase/collections)
+  --source {NCBI}       Source of data (default: NCBI)
+  --threads THREADS     Number of threads
+  --overwrite           Overwrite existing files
   --exclude_name_substring EXCLUDE_NAME_SUBSTRING [EXCLUDE_NAME_SUBSTRING ...]
```

Comment:
I think we should make `['cds_from', 'rna_from', 'ERR']` a default `exclude_name_substring` for GTDB collection.

Reply:
in NCBI, I have seen `xxx. cds_from_genomic.fna.gz`, `xxx.rna_from_genomic.fna.gz` and `xxx.ERR_genomic.fna.gz`. Those are the files I think we should by default exclude.

Reply:
Just found out we have something defined here. Maybe just move it to a common module. 

https://github.com/kbase/collections/blob/main/src/loaders/compute_tools/tool_common.py#L48

Reply:
Updated!